azure-iot-hub-devicesdk/tests/test_message.py

```diff
@@ -0,0 +1,76 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.iot.hub.devicesdk.message import Message
+import pytest
+from datetime import datetime
+
+
+def test_construct_message_with_string():
+    s = "After all this time? Always"
+    time = datetime.utcnow()
+    mes = Message(s, time)
```

Comment:
>mes [](start = 4, length = 3)

Nit: This shortening of message reads funny to me.  I've always seen it as msg, which I see you use for a string below.